x-pack/legacy/plugins/canvas/public/components/datasource/datasource_preview/datasource_preview.js

```diff
@@ -19,16 +20,32 @@ import {
 import { Datatable } from '../../datatable';
 import { Error } from '../../error';
 
+import { ComponentStrings } from '../../../../i18n';
+const { DatasourceDatasourcePreview: strings } = ComponentStrings;
+
 export const DatasourcePreview = ({ done, datatable }) => (
   <EuiOverlayMask>
     <EuiModal onClose={done} maxWidth="1000px" className="canvasModal--fixedSize">
       <EuiModalHeader>
-        <EuiModalHeaderTitle>Datasource Preview</EuiModalHeaderTitle>
+        <EuiModalHeaderTitle>{strings.getModalTitle()}</EuiModalHeaderTitle>
       </EuiModalHeader>
       <EuiModalBody className="canvasDatasourcePreview">
         <EuiText size="s" color="subdued">
           <p>
-            Click <strong>Save</strong> in the sidebar to use this data.
+            <FormattedMessage
+              id="xpack.canvas.datasourceDatasourcePreview.modalDescription"
+              defaultMessage="Click {saveLabel} in the sidebar to use this data."
+              values={{
+                saveLabel: (
+                  <strong>
+                    <FormattedMessage
+                      id="xpack.canvas.datasourceDatasourcePreview.modalSaveDescription"
+                      defaultMessage="Save"
```

Comment:
I'm not sure if it'll be ambiguous to the translator what the "Save" is referring to, but it might be nice to add a description here. 

Also, can we reuse `DatasourceDatasourceComponent.getSaveButtonLabel` here? I doubt the label for the button will change, but since we are referring to a button elsewhere, we can keep them in sync if it does change.